libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/item/VcStoreItem.java

```diff
@@ -24,6 +24,7 @@ public class VcStoreItem implements DynamodbItem {
     private String credential;
     private Instant dateCreated;
     private Instant expirationTime;
+    private Instant migrated;
```

Comment:
(I assume this will correctly parse as `null` for existing records with no value)

Reply:
Yep yep